.vitest/src/paymaster.ts

```diff
@@ -124,7 +116,7 @@ export async function deployPaymasterContract(
     getPaymasterDetails({
       implAddress,
       // for now we just support 0.6.0
-      entrypoint: "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789",
+      entrypoint: entrypoint060,
```

Comment:
plans to support 0.7 in the future?

Reply:
definitely! I'll add it as a I need it. I think I'll need it to test multi-owner LA so it will come in that PR. With that, I can also test ALL versions of LA in the e2e test rather than forcing it to work on v1.1.0 only which will be more coverage than today too.

Overall goal is to have 1:1 parity with today's tests and then add the improvements that this set up enables us to introduce